src/vs/editor/contrib/unicodeHighlighter/browser/unicodeHighlighter.ts

```diff
@@ -510,6 +510,10 @@ export class UnicodeHighlighterHoverParticipant implements IEditorHoverParticipa
 	}
 }
 
+function fixedEncodeURIComponent(str: string): string {
+	return encodeURIComponent(str.replace(/[()]/g, escape));
```

Comment:
This double-escapes the string though.

![image](https://user-images.githubusercontent.com/2931520/160625727-b17cc2b3-fe48-4740-923a-810fb235961a.png)

Reply:
Updated.